src/resources/views/crud/buttons/create.blade.php

```diff
@@ -1,3 +1,8 @@
 @if ($crud->hasAccess('create'))
-	<a href="{{ url($crud->route.'/create') }}" class="btn btn-primary" data-style="zoom-in"><span class="ladda-label"><i class="la la-plus"></i> {{ trans('backpack::crud.add') }} {{ $crud->entity_name }}</span></a>
+	<a href="{{ url($crud->route.'/create') }}" class="btn btn-primary" data-style="zoom-in">
+		<span class="ladda-label">
```

Comment:
Ugh I see there's still some leftover `ladda` classes over here. But AFAIK we don't load [ladda buttons anymore](https://lab.hakim.se/ladda/). If it's not onpage I think we should remove the classes too.

Reply:
I was going to ask you about this `ladda` classes 😅
Glad we can clean that too 😅